corehq/toggles.py

```diff
@@ -220,14 +221,18 @@ def randomness_percent(self):
     def _get_identifier(self, item):
         return '{}:{}:{}'.format(self.namespaces, self.slug, item)
 
-    def enabled(self, item, **kwargs):
+    def enabled(self, item, namespace=Ellipsis):
+        if namespace is Ellipsis:
+            logging.warning('PredictablyRandomToggle.enabled() may be invalid if namespace is not set')
```

Comment:
I think most violations of this will never be noticed. Consider raising an exception instead.

Reply:
This class currently only gets used in one place at the moment, so I think that's a good suggestion. (And if throwing exceptions is potentially a problem, a soft assert might be a reasonable compromise.)